entrypoint.sh

```diff
@@ -169,7 +169,8 @@ main() {
       -passout "pass:${keystore_pass}" \
         >/dev/null 2>&1
 
-  touch server/startup/newinstall
+  # Removed as this setting results in every startup being interrupted as a boostrap
+  #touch server/startup/newinstall
```

Comment:
Is there a reason to keep this as a comment, or can we delete it completely?

Reply:
removed